src/ApiManagement/ApiManagement.ServiceManagement/Commands/NewAzureApiManagementResourceLocation.cs

```diff
@@ -0,0 +1,79 @@
+﻿//  
+// Copyright (c) Microsoft.  All rights reserved.
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+using System.Collections;
+using System.Linq;
+
+namespace Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Commands
+{
+    using Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Models;
+    using System.Management.Automation;
+
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ApiManagementResourceLocation")]
```

Comment:
As `PsApiManagementResourceLocation` is a in-memory object, it's better to rename `New-AzureApiManagementResourceLocation` to `New-AzureApiManagementResourceLocationObject`.

Reply:
cmdlet renamed